src/components/grocery-view/grocery-section/CategoryRow.js

```diff
@@ -2,32 +2,46 @@
 import React, { useState } from 'react';
 
 // Components
-import { Grid, Typography, IconButton } from '@material-ui/core';
+import { Typography, IconButton } from '@material-ui/core';
 import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
 import ChevronRightIcon from '@material-ui/icons/ChevronRight';
 import GroceryItem from './GroceryItem';
 
 // Styles
 import styles from '../styles/category-row.module.css';
 
-// REFACTOR: Use real data here
-const ITEMS = ['broccoli', 'corn', 'bread', 'milk', 'beef', 'cheese'];
+// Data
+import foodData from '../../../../content/food-data.json';
 
-const CategoryRow = ({ category }) => {
+const CategoryRow = ({ category, cartState, setCartState }) => {
   const [startIndex, setStartIndex] = useState(0);
-  // REFACTOR: Using fake data right now
-  const groceryItems = ITEMS.map((item) => (
-    <GroceryItem key={item} item={item} />
-  ));
+
+  const groceryItems = Object.keys(cartState).map((itemKey) => {
+    if (foodData[itemKey].other.category.includes(category)) {
+      return (
+        <GroceryItem
+          cartState={cartState}
+          setCartState={setCartState}
+          key={itemKey}
+          item={itemKey} />
+      );
+    }
+    return null;
+  }).filter((isNotNull) => {
+    if (isNotNull) {
+      return true;
+    }
+    return false;
```

Comment:
I'm pretty sure this could be simplified to `.filter((isNotNull) => isNotNull)` since a null value is falsy